legal-api/report-templates/annualReport.html

```diff
@@ -260,21 +269,13 @@
       <tfoot>
         <tr class="bcgov-footer-row bcgov-blue-bg">
           <td></td>
-          <td colspan="2">
-            <span>BC Registries and Online Services</span>
-          </td>
-          <td class="align-right">
-            {{ environment }}
-          </td>
+          <td colspan="2"><span>BC Registries and Online Services</span></td>
```

Comment:
You might have had an old file - this span isn't needed.

Reply:
You're right. I fixed my base and now it's OK!